### ISSUE TITLE
Throw "Can not find limit span" recently

### ISSUE THREAD

**Issue by djfjeff** [titilambert/pyebox#7]
Hi,

It seems that pyebox always throw "Can not find limit span" recently when run, both standalone and in Home Assistant : 

```
pyebox -u XXX -p XXX

Can not find limit span
```

Thanks!

**Comment by rouiz** [comment]
Same here.
Error on receive last EBox data: Can not find limit span
HA version : 0.96+

@djfjeff  Did you solve it?
thanks

> Hi,
> 
> It seems that pyebox always throw "Can not find limit span" recently when run, both standalone and in Home Assistant :
> 
> ```
> pyebox -u XXX -p XXX
> 
> Can not find limit span
> ```
> 
> Thanks!

**Comment by djfjeff** [comment]
@rouiz I did a quick and dirty modification of the client.py file to fix the error message. I was not using the cash balance, etc. I am only using the download, upload and total, so this is working for me.

I will try to take a look if I have time to fix it properly but I cannot guarantee anything on this.

Rename the attached file to client.py and replace the default one with it.

[client.txt](https://github.com/titilambert/pyebox/files/3529833/client.txt)

**Comment by rouiz** [comment]
@djfjeff excellent. I'll try that. Thanks for the help.

**Comment by gjacquet** [comment]
Yup, looks like limit and usage are no longer loaded by default on the home page 😞 
I believe I have fix for this in #8 

That would be nice to have it fixed as I would love to setup high usage alerts!

**Comment by bullmoose20** [comment]
Continuously get the "Can not find limit span" issue in Home Assistant. Has this been fixed yet?
